frontend/src/app/components/op-context-menu/op-context-menu.html

```diff
@@ -5,8 +5,8 @@
       role="menu"
       [ngClass]="{'-empty': items.length === 0 }">
     <ng-container *ngFor="let item of items">
-      <li *ngIf="item.divider && !item.hidden" class="dropdown-divider"></li>
-      <li *ngIf="!item.divider && !item.hidden">
+      <li *ngIf="item.divider" class="dropdown-divider"></li>
```

Comment:
@aleixsuau What has this todo with your PR? Did you come across this with Oliver? So are you sure it is safe to delete?